timetable/utils.py

```diff
@@ -51,8 +51,10 @@ def get_timetable_context(lessons):
     table = OrderedDict()
     for period in periods:
         table[period] = (period_strs[period], OrderedDict())
-        for alt_period in alternative_periods.filter(number=period):
-            table[period] = table[period] + (alt_period,)
+        for dayPlan in DayPlan.objects.all():
+            if dayPlan.is_today and not dayPlan.schedule.is_default:
+                for alt_period in alternative_periods.filter(number=period):
+                    table[period] = table[period] + (alt_period,)
```

Comment:
When you've got more than one non-default schedule, the system displays them all.
![image](https://github.com/zorzalo3/zorza/assets/44808028/6c7198ec-a9bc-4cee-8b3f-1e484c300b57)
![image](https://github.com/zorzalo3/zorza/assets/44808028/d4cbeb9b-1c27-417d-aa0a-019b9baae377)

Reply:
There is no point in displaying multiple (more than default + alternative assigned for today) periods in the timetable. More details you can find in the issue. https://github.com/zorzalo3/zorza/issues/4#issuecomment-1838217936